### ISSUE TITLE
Boot from hard drive fails - No configuration file found

### ISSUE THREAD

**Issue by IgorWolbers** [Drive-Trust-Alliance/sedutil#207]
Let me start by saying thank you for this software, this is really the only option most of us have if we want to enable opal encryption on our drive(s). I really appreciate the time and effort you have put into this project!

----

I enabled opal on my hard drive successfully but I can't boot off my hard drive to the pass phrase unlock screen, I can only boot off a USB stick with the `UEFI64` image burned on it (which allows me to enter my password phrase and unlock my drive) after which the drive does boot windows without a problem. 

I get the following prompt when booting off the hard drive:

	WARNING: No configuration file found
	boot:

It seems that `boot` is a prompt, I can enter text on the screen although I have no idea what I should enter at this point as this is an unexpected screen.

I have followed the steps exactly as described in the [Encrypting your drive](https://github.com/Drive-Trust-Alliance/sedutil/wiki/Encrypting-your-drive) page. In the BIOS I have also compared the shadow boot folder to the one on the USB drive and it shows the same files (I have not been able to verify the content of these files).

These are my specs and settings, any pointers would be most appreciated. I have searched the existing issues on this site but could not find a solution. If there is anything else I can provide please let me know.	
	
* Dell laptop: Inspiron 5770. BIOS version 1.1.1 (*latest*) 
* BIOS Options: 
  * AHCI = On 
  * Boot/Startup = Thorough
  * Secure Boot = Disabled
* Hard drive: 
  * Samsung SSD 960 PRO 512GB (NVME)
  * Firmware: 4B6QCXP7 (*latest*) 
* SedUtil 
  * Version: 1.15.1 (*latest*) 
  * Image loaded: UEFI64.img.gz

**Comment by cristim** [comment]
This is a common problem, I had it as well. The UEFI boot menu entry needs to be edited from the system setup menu by browsing through the PBA content

**Comment by IgorWolbers** [comment]
I added a second boot entry to the path

    \EFI\boot\bootx64.efi

This fixed the problem. I am not sure why the original entry, which is still there, did not work as this was listed as pointing to the same file. So I have 2 entries now, the existing one and this new one.

Thank you @cristim.

**Comment by r0m30** [comment]
This is an artifact of how different system bios' search for bootable media.  Sedutil uses the default EFI partition layout and names but not all vendors search for that entry on internal storage, I wish the EFI partition was formatted with a filesystem that supports links then we could create multiple structures to work around the different vendor implementations without having to make the PBA huge.

**Comment by ranshalit** [comment]
Hello,
If I may please ask on the subject.
In some issues of boot it is suggested to boot from usb and unlock the driver.
I understand how to boot from usb, but how do we then unlock the driver ?
Is it by typing linuxpba with "debug" password. Is this the action that unlock the driver so that we can then read its content ?

Thank you!
Ran

**Comment by IgorWolbers** [comment]
@ranshalit - It would be best to open a new issue / question instead. Asking an additional, and possibly unrelated question, on a closed issue is not a good idea.

**Comment by ranshalit** [comment]
OK.

**Comment by MateoOO7** [comment]
I have a similar situation to the author of the thread. PBA on USB and the same error. I tried to add an entry to the EFI partition on my disk like the author with the command:

 `sudo efibootmgr -c -d /dev/sda -p 1 -L “New-entry” -l '\EFI\boot\bootx64.efi'`

but it doesn't work. The error still occurs. I've tried many ways to work around it, but to no result.

**Comment by MateoOO7** [comment]
I have a similar situation to the author of the thread. PBA boot from USB and the same error. I tried to add an entry to the EFI partition on my disk like the author with the command:

`sudo efibootmgr -c -d /dev/nvme0n1 -p 1 -L “New-entry” -l '\EFI\boot\bootx64.efi`

but it doesn't work. The error still occurs. I've tried many ways to work around it for example change syslinux.cfg file in PBA, but witch no result. Im using Version: 1.15.1 PBA.

UPDATE:
I fix it by single switch... I set FASTBOOT to disable in BIOS...
